docs/docsite/rst/user_guide/playbooks_filters.rst

```diff
@@ -1660,6 +1660,8 @@ To get a date object from a string use the `to_datetime` filter::
 
     # get amount of days between two dates. This returns only number of days and discards remaining hours, minutes, and seconds
     {{ (("2016-08-14 20:00:12" | to_datetime) - ("2015-12-25" | to_datetime('%Y-%m-%d'))).days  }}
+    
+.. note:: To get all string possibilities, check https://docs.python.org/3/library/datetime.html#strftime-and-strptime-behavior
```

Comment:
This probably needs some "word smithing" from our docs team.  I'd also recommend we indicate "python date format strings", possibly without `python`.

Reply:
> This probably needs some "word smithing" from our docs team. I'd also recommend we indicate "python date format strings", possibly without `python`.

Well, I just copied the wording from the next note some lines below and changed the url.

Reply:
Thanks @torvitas for opening this PR to enrich the Ansible documentation. 

From the link you added, it looks like the `strftime` method exists on both `datetime` and `time`, are there differences in implementation between the two? Either way, I'd support mentioning the full docs in both places, since most docs users don't read the whole page, they scan for the section they need and read that. 

Copying the text from the next entry gives the docs a nice consistency, but the existing phrasing (which I might have written) isn't great. How about:

```suggestion
.. note:: For a full list of format codes for working with python date format strings, see https://docs.python.org/3/library/datetime.html#strftime-and-strptime-behavior.
```